code/modules/cargo/expressconsole.dm

```diff
@@ -48,35 +50,22 @@
 		message = "(&!#@ERROR: ROUTING_#PROTOCOL MALF(*CT#ON. $UG%ESTE@ ACT#0N: !^/PULS3-%E)ET CIR*)ITB%ARD."
 	
 	data["message"] = message
-
-	if (emagged)  
-		for(var/pack in SSshuttle.supply_packs)
-			var/datum/supply_pack/P = SSshuttle.supply_packs[pack]
-			if (P.name == "Toy Crate")//Can only order toys if emagged. You gotta spend 10K points to crash a droppod somewhere on the station
-				data["supplies"][P.group] = list(
-							"name" = P.group,
-							"packs" = list()
-						)
-				data["supplies"][P.group]["packs"] += list(list(
-						"name" = P.name,
-						"cost" = P.cost * 2, //displays twice the normal cost
-						"id" = pack
-					))		
-	else
-		for(var/pack in SSshuttle.supply_packs)
-			var/datum/supply_pack/P = SSshuttle.supply_packs[pack]
-			if(!data["supplies"][P.group])
-				data["supplies"][P.group] = list(
-					"name" = P.group,
-					"packs" = list()
-				)
-			if((P.hidden) || (P.contraband) || (P.special))//no fun allowed
-				continue
-			data["supplies"][P.group]["packs"] += list(list(
-				"name" = P.name,
-				"cost" = P.cost * 2, //displays twice the normal cost
-				"id" = pack
-			))
+	for(var/pack in SSshuttle.supply_packs)

```

Comment:
this should be created elsewhere, not fresh in every ui_data

Reply:
since this can be different per machine do the list creation in initialize via a proc so admins can call it if they wish to manually change the list